shell/platform/fuchsia/flutter/meta/flutter_aot_product_runner.cmx

```diff
@@ -26,6 +26,7 @@
             "fuchsia.ui.input.ImeService",
             "fuchsia.ui.input3.Keyboard",
             "fuchsia.ui.scenic.Scenic",
+            "fuchsia.ui.pointerinjector.Registry",
```

Comment:
We will need to change the product session to route this service to flutter_runner in addition to the flutter app, until then connections will fail.  I would go ahead and land that before landing this, I can show you how to do it offline

Reply:
Removed the CML/CMX changes for now. Will add them back in a follow up CL when the changes mentioned in the comment have been made